test/src/test/java/hudson/model/labels/LabelAtomSecurity1986Test.java

```diff
@@ -194,15 +169,11 @@ public void programmaticCreation() throws IOException {
     @Test
     @Issue("SECURITY-1986")
     public void startsWithTripleDotBackslash() throws Exception {
-        try {
-            j.submit(j.createWebClient().goTo("labelAtom/...%2ffoo/configure").getFormByName("config"));
-            fail("Should have rejected label.");
-        } catch (FailingHttpStatusCodeException e) {
-            assertThat(e.getStatusCode(), is(HttpURLConnection.HTTP_BAD_REQUEST));
-            LabelAtom foo = j.jenkins.getLabelAtom(".../foo");
-            XmlFile configFile = foo.getConfigFile();
-            assertFalse(configFile.getFile().exists());
-        }
+        FailingHttpStatusCodeException e = assertThrows("Should have rejected label.", FailingHttpStatusCodeException.class, () -> j.submit(j.createWebClient().goTo("labelAtom/...%2ffoo/configure").getFormByName("config")));
+        assertThat(e.getStatusCode(), is(HttpURLConnection.HTTP_BAD_REQUEST));
+        LabelAtom foo = j.jenkins.getLabelAtom(".../foo");
+        XmlFile configFile = foo.getConfigFile();
+        assertFalse(configFile.getFile().exists());
```

Comment:
Out of scope, but looks like low-hanging fruit for deduplication

Reply:
Indeed